packages/next-plugin-treat/index.js

```diff
@@ -0,0 +1,19 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const TreatPlugin = require('treat/webpack-plugin');
+
+module.exports = (pluginOptions = {}) => (nextConfig = {}) => {
```

Comment:
You may also want to call "maybe" passed in `nextConfig.webpack`, so chaining still works.

https://github.com/zeit/next.js/blob/canary/packages/next-mdx/index.js#L17-L19